src/ui/components/ServerLaunchSection.tsx

```diff
@@ -314,6 +318,7 @@ export default class ServerLaunchSection extends React.Component<IProps, IState>
               <img className="ExtrasIcon" id="extrasIcon" src={Plus} />
             </BigButton>
           )}
+
```

Comment:
?

Reply:
Editing mistake, thank you for letting me know. I will keep on eye on those additional lines now.